Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN pip install docutils==0.17.1 # Need to pin docutils to an old version for no
 RUN pip install referencing
 RUN pip install jupyter-book
 # Pinning pandas until altair 5.1.2 to avoid future warning https://github.com/altair-viz/altair/issues/3181
-RUN pip install numpy jinja2 pandas"<2.1" altair">=5.1.1" "vegafusion[embed]" vl-convert-python">=0.13" click ibis-framework ghp-import jupytext nodejs lxml
+RUN pip install numpy jinja2 pandas altair">=5.1.2" "vegafusion[embed]" vl-convert-python">=0.14" click ibis-framework ghp-import jupytext nodejs lxml
```

Comment:
the vl-convert bump is needed to go with altair 5.1.2 and the pandas restriction is no longer necessary as that warning was removed in 5.1.2